data/fixtures/scopes/textual/boundedParagraph4.scope

```diff
@@ -0,0 +1,44 @@
+aaa(bbb) {
+  ccc
+}
+---
+
+[#1 Content] =
+[#1 Removal] =
+[#1 Domain] = 0:0-2:1
+  >----------
+0| aaa(bbb) {
+1|   ccc
+2| }
+   -<
+
+[#1 Insertion delimiter] = "\n\n"
+
+
+[#2 Content] =
+[#2 Removal] =
+[#2 Domain] = 0:4-0:7
+      >---<
+0| aaa(bbb) {
```

Comment:
I would be tempted to remove this one? Seems odd to get trapped inside parens on a single line when you say "short block", but maybe we should see how it feels in practice?